designer/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentServiceImpl.scala

```diff
@@ -486,7 +540,7 @@ class DeploymentServiceImpl(
       .getOrElse(state)
   }
 
-  override def markActionExecutionFinished(processingType: ProcessingType, actionId: ProcessActionId)(
```

Comment:
Why do you remove processing type checking? Currently, one DeploymentManager that is configured in some 
 processing type can modify actions maintained by another DeploymentManager

Reply:
It does not prevent modifying "not-my-action" when using "not-my-action-id" with "not-my-processing-type" either. 

That validation does not collide with action-to-customaction alignment, I excluded from this PR.